content/components/toast-update.mdx

```diff
@@ -0,0 +1,88 @@
+
+### Usability and accessibility concerns with toasts
+WCAG compliant toasts are often missed by users for a variety of reasons: 
+- Ultrawide monitor
+- Looking away from the computer screen for the duration of an auto-dismissing toast
+- Assistive Technology reliability or setting for announcing aria-live components
+- Zoomed in view
+- Zoomed in view requires that focus is brought to a toast in order to be seen by someone using zoom on a screen. This can be distruptive to a user's orentiation of the screen, especially if not critial to their workflow.
+- Toast fatigue (used too often for things that aren't important to a user's workflow)
+
+### Tiny history lesson
+Toasts started in the mobile space, where the intended use cases make more sense than on a desktop. 
+- Some situations have successful interactions that then present an undo option to the action you've just performed, like a sent email message. It's a lot easier to perform an undo action on a desktop with the availability of keyboard shortcuts to undo actions than it is on mobile, whose keyboards are often hidden unless in an active text editor, and don't contain modifier keys. 
+- Before Android coined Toasts, iOS created push notifications whose use case was to notify of non-critical new things in other areas of the product to explore at a user's leisure. 
+- Before Android toasts and iOS push notifications, we were notified "You've got mail" when receiving a new email message from AOL on a desktop computer, which lead to subsequent overuse of dialogs and notifications.
+
+### Our use cases
+GitHub has its own system for notifying our users of new or updated activity to be explored at their leisure in other parts of the product, GitHub notifications. With those use cases covered, we're left with a need to notify users of the status of actions they've taken on a page in the product to help them successfully complete their tasks.
+
+## Alternatives to toasts
+Rather than reach for a toast, we've found that there are better patterns to use in the majority of our use cases.
+
+/*Scenarios*/
+### 1. Successful actions
+Excessive use of successful notification messages can indicate product instability. If we have to really let users know every time something was successful, we're not building a reliable product.
+
+If the change made on the page is visible to the user, we can rely on the user's ability to see the change and know that it was successful. 
+
+For example, if a user activates a toggle control, and the toggle control doesn't flip back to the original postion, a user should be confident that the change has occured with the absense of an error message. Likewise for screen readers, after activating the toggle control the control should read out the current control's state. 
```

Comment:
I agree with this. But maybe we do need to look into our [saving docs](https://primer.style/ui-patterns/saving) and add some standard checkmark indicator. 

Personally, I find when you have a page with a save button and part of it is auto-saving, this is hard to trust, if you have no indicator.

A simple inline message "✅ changes saved" can help here.